### DISCUSSION TITLE
KMPNotifier Desktop Support issues?

### DISCUSSION THREAD

**Discussion by mirzemehdi** [mirzemehdi/KMPNotifier#47] [General]
Please share any issues that occurred related to KMPNotifier Desktop part as it can be considered in alpha state for desktop target. For now Desktop supports only local notifications only, push notifications are not available

**Comment by N7ghtm4r3** [comment]
Hi, I' m using your library to implement the notifications on a Desktop application and my answer is can I handle this also if the `Notifier` is the local one, if yes where I need to place the following snippet to make it works? Thank you 

```kotlin
NotifierManager.addListener(object : Listener {
  override fun onNotificationClicked(data: PayloadData) {
      super.onNotificationClicked(data)
      // handle payload
  }
})
```

**Reply by mirzemehdi** [reply]
@N7ghtm4r3 Listeners are not available for Desktop part, currently

**Reply by N7ghtm4r3** [reply]
Thank you for the info! It is planned their implementation also for Desktop?

**Reply by mirzemehdi** [reply]
tbh, currently I don't have much enough knowledge in that area. So can't say it rn
